docs/rsdl/rapid-pro-rsdl-intro.md

```diff
@@ -3,80 +3,238 @@ id: rsdl-intro
 title: RAPID SDL intro
 ---
 
-# Introduction to RAPID Pro schema definition language (RSDL)
+# Introduction to RAPID Schema Definition Language (RSDL)
 
-> DRAFT
+RAPID Schema Definition Language (RSDL) is language to describe Web APIs.
 
-RAPID Pro schema definition language (RSDL) is language to describe Web APIs.
-RSDL is based on a [profile](<https://en.wikipedia.org/wiki/Profile_(engineering)>) of the
-[OData](https://en.wikipedia.org/wiki/Open_Data_Protocol) specification with the goal to provide an easy way
-to create a Web API that is compatible with OData and can evolve into a more elaborate version.
+RSDL is based on the [RAPID PROfile](<https://en.wikipedia.org/wiki/Profile_(engineering)>) of the
+[OData](https://en.wikipedia.org/wiki/Open_Data_Protocol) specification. RAPID provides an easy way
+to envision, create, and consume a Web API that is compatible with the OData Standard and can evolve over time to support more advanced scenarios.
 
 ## Introductory Example
 
-The core description of the "shape" of the API is given by a RSDL document.
+RAPID APIs are defined by a schema, which can easily be described through RSDL.
 
-Below is a first example of an RSDL file that describes an API of a service that provides data on companies and their employees
+### Defining a Type
+Let's say that we wanted our API to deal with information about a company. We could describe the properties of a company as follows:
 
-```
+~~~rsdl
 type company
 {
-    @key stockSymbol: string
+    stockSymbol: string
     name: string
     incorporated: dateTime
-    employees: [employee]
 }
+~~~
+
+Our company has three properties; a stockSymbol, a name, and the date of incorporation.
+
+Properties in RAPID can be string, integer, boolean, double, decimal, or dateTime primitive types. They can also be an [enum type](#defining-an-enums), a [declared type](#defining-a-nested-property), or a collection of any of the above.
+
+### Defining a Service
+Now we can create a service named "jetsons" that returns information about our company:
+
+~~~rsdl
+jetsons {
```

Comment:
Will add an issue to change the grammar and tool to allow naming services.

Reply:
Actually, I don't think the name "jetsons" is supported -- it should just be "service".

However, this may change depending on if/how we introduce namespaces.

Reply:
You're right, just "service" instead of "jetsons"